packages/frontend/src/utils/url.js

```diff
@@ -0,0 +1,16 @@
+export const buildUrlOptParam = (url, params = {}) => {
```

Comment:
We usually use [`stringify` from the query-string library](https://github.com/sindresorhus/query-string#stringifyobject-options) to build these kinds of request payloads. This may be useful here?